pkg/skaffold/deploy/cloudrun/deploy_test.go

```diff
@@ -77,6 +80,17 @@ func TestDeploy(tOuter *testing.T) {
 					Namespace: "my-project",
 				},
 			},
+			errCode: proto.StatusCode_DEPLOY_CLOUD_RUN_GET_SERVICE_ERR,
```

Comment:
can you also add a test showing the precedence eg: the skaffold.yaml value overwriting the cloudrun.yaml value?

Reply:
"test deploy with specified project" does that already.

Reply:
Ah I see, missed the value change in the diff.  Merging now